helm/templates/config.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     {{- include "zaakafhandelcomponent.labels" . | nindent 4 }}
 data:
+  _JAVA_OPTIONS: {{ .Values.javaOptions | default "-Xmx1024m -Xms1024m -Xlog:gc::time,uptime" | quote }}
```

Comment:
That would reset the current 1700MB to 1024MB right?

Reply:
correct, that would have to be set via the values then